.github/workflows/examples-nrfconnect.yaml

```
@@ -84,9 +84,12 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh 'pip3 install -r scripts/setup/requirements.nrfconnect.txt'
                   scripts/run_in_build_env.sh "./scripts/tools/nrfconnect/tests/test_generate_factory_data.py"
-            - name: Run unit tests for Zephyr native_posix_64 platform
+            - name: Run unit tests for Zephyr native_posix_64 platform with NVS
               run: |
                   scripts/run_in_build_env.sh "./scripts/build/build_examples.py --target nrf-native-sim-tests build"
+            - name: Run unit tests for Zephyr native_posix_64 platform with ZMS
+              run: |
+                  scripts/examples/nrfconnect_example.sh test_driver native_sim -DCONFIG_ZMS=y -DCONFIG_ZMS_LOOKUP_CACHE=y -DCONFIG_ZMS_LOOKUP_CACHE_SIZE=512 -DCONFIG_ZMS_LOOKUP_CACHE_FOR_SETTINGS=y -DCONFIG_NVS=n
```

```suggestion
                  scripts/examples/nrfconnect_example.sh ../src/test_driver native_sim -DCONFIG_ZMS=y -DCONFIG_ZMS_LOOKUP_CACHE=y -DCONFIG_ZMS_LOOKUP_CACHE_SIZE=512 -DCONFIG_ZMS_LOOKUP_CACHE_FOR_SETTINGS=y -DCONFIG_NVS=n
```
would it help? As the script expects that you build an example.

---

Yes, I've tested it this was locally in command line, but then I've forgotten to update the file :)

---

also, you need to run `zephyr.exe` manually after you build